glide.yaml

```diff
@@ -7,7 +7,7 @@ import:
   - time
 
 - package: github.com/m3db/m3cluster
-  version: 53fc512c11e1ed03db2d65dac4c139a3c2ff2eda
+  version: ebb48b02b8cf382df42062179dc29ce6b1331180
```

Comment:
Do you need to update this?

Reply:
Yeah I fixed a bug in the m3cluster implementation of the in-memory KV store to actually return kv.ErrConditionCheckFailed when there was a failed condition check. Without this we would need to mock a KV implementation for tests.

Reply:
Here is the PR in question:
https://github.com/m3db/m3cluster/pull/218